x-pack/plugins/data_enhanced/config.ts

```diff
@@ -12,7 +12,8 @@ export const configSchema = schema.object({
       enabled: schema.boolean({ defaultValue: false }),
       pageSize: schema.number({ defaultValue: 10000 }),
       trackingInterval: schema.duration({ defaultValue: '10s' }),
-      inMemTimeout: schema.duration({ defaultValue: '1m' }),
+      completedTimeout: schema.duration({ defaultValue: '5m' }),
```

Comment:
Just to confirm I understand correctly: 
This is a timeout which determines for how long user can still save the session after it is completed? 
And we need to use it client side to disable "saving" (I can add this in separate pr)

Reply:
Yep. Bingo.

Reply:
Actually, to clarify - the timeout is from the creation time.
Maybe the previous "onScreenTimeout" was better?